.azure/gpu-pipeline.yml

```
@@ -18,6 +18,15 @@ pr:
 
 jobs:
   - job: pytest
+    strategy:
+      matrix:
+        'PyTorch - stable':
+          PyTorch: '1.8.1'
+          CUDA: '11.1'
+        'PyTorch - latest':
```

Not a strong opinion, but should we maybe use "LTS" and "stable" instead?

```suggestion
        'PyTorch - LTS':
          PyTorch: '1.8.1'
          CUDA: '11.1'
        'PyTorch - stable':
```

ref: https://pytorch.org/docs/versions.html

---

good pints! thx :)